### PR TITLE
Remove ParseQueueConnStr as it is unused

### PR DESCRIPTION
**Pull request by zeripath**
Remove ParseQueueConnStr as `modules/nosql` has taken over all of its functions.

Signed-off-by: Andrew Thornton <art27@cantab.net>

### CODE REVIEW

File: modules/setting/queue.go [hunk 1]
```diff
@@ -22,12 +20,8 @@ type QueueSettings struct {
 	BatchLength      int
 	ConnectionString string
 	Type             string
-	Network          string
```
Comment (6543) [inline modules/setting/queue.go:25]: do we have reverences in example.ini or ohter docs left?

Comment (zeripath) [reply modules/setting/queue.go:25]: Nope.

**Review by 6543 — APPROVED** [review]

**Comment by lunny** [comment]
> Remove ParseQueueConnStr as `modules/nosql` has taken over all of its functions.
> 
> Signed-off-by: Andrew Thornton [art27@cantab.net](mailto:art27@cantab.net)

Isn't an old configuration will be break here?

**Comment by zeripath** [comment]
> > Remove ParseQueueConnStr as `modules/nosql` has taken over all of its functions.
> > Signed-off-by: Andrew Thornton [art27@cantab.net](mailto:art27@cantab.net)
> 
> Isn't an old configuration will be break here?

Nope ToRedisURI handles all of that.

**Review by lunny — APPROVED** [review]

**Comment by 6543** [comment]
:rocket:

**Pull request merged**